examples/cloud-operations/adfs/main.tf

```diff
@@ -0,0 +1,200 @@
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+locals {
+  prefix = (var.prefix == null || var.prefix == "") ? "" : "${var.prefix}-"
+}
+
+module "project" {
+  source = "../../../modules/project"
+  billing_account = (var.project_create != null
+    ? var.project_create.billing_account_id
+    : null
+  )
+  parent = (var.project_create != null
+    ? var.project_create.parent
+    : null
+  )
+  prefix = var.project_create == null ? null : var.prefix
+  name   = var.project_id
+  services = [
+    "compute.googleapis.com",
+    "dns.googleapis.com",
+    "managedidentities.googleapis.com"
+  ]
+  policy_list = {
+    "constraints/compute.restrictLoadBalancerCreationForTypes" = {
+      inherit_from_parent = null
+      suggested_value     = null
+      status              = true
+      values              = ["EXTERNAL_HTTP_HTTPS"]
+    }
+  }
+}
+
+module "vpc" {
+  count      = var.network_config == null ? 1 : 0
+  source     = "../../../modules/net-vpc"
+  project_id = module.project.project_id
+  name       = "${local.prefix}vpc"
+  subnets = [
+    {
+      ip_cidr_range      = var.subnet_ip_cidr_block
+      name               = "subnet"
+      region             = var.region
+      secondary_ip_range = null
+    }
+  ]
+}
+
+resource "google_active_directory_domain" "ad_domain" {
+  project             = module.project.project_id
+  domain_name         = var.ad_dns_domain_name
+  locations           = [var.region]
+  authorized_networks = [module.vpc[0].network.id]
+  reserved_ip_range   = var.ad_ip_cidr_block
+}
+
+module "server" {
+  source        = "../../../modules/compute-vm"
+  project_id    = module.project.project_id
+  zone          = var.zone
+  name          = "adfs"
+  instance_type = var.instance_type
+  network_interfaces = [{
+    network    = var.network_config == null ? module.vpc[0].self_link : var.network_config.network
+    subnetwork = var.network_config == null ? module.vpc[0].subnet_self_links["${var.region}/subnet"] : var.network_config.subnet
+    nat        = false
+    addresses  = null
+  }]
+  metadata = {
+    # Enables OpenSSH in the Windows instance
+    sysprep-specialize-script-cmd = "googet -noconfirm=true update && googet -noconfirm=true install google-compute-engine-ssh"
+    enable-windows-ssh            = "TRUE"
+    # Set the default OpenSSH shell to Powershell
+    windows-startup-script-ps1 = <<EOT
+      New-ItemProperty -Path "HKLM:\SOFTWARE\OpenSSH" `
+      -Name DefaultShell `
+      -Value "C:\Windows\System32\WindowsPowerShell\v1.0\powershell.exe" `
+      -PropertyType String `
+      -Force
+    EOT
+  }
+  service_account_create = true
+  service_account_scopes = ["https://www.googleapis.com/auth/cloud-platform"]
+  boot_disk = {
+    image = var.image
+    type  = var.disk_type
+    size  = var.disk_size
+  }
+  group = {
+    named_ports = {
+      http = 443
+    }
+  }
+  tags = [
```

Comment:
Can you put small lists on a single line, here and in other places (eg ssl certificates config domains below)